README.md

```diff
@@ -26,8 +26,11 @@ let jfrogClient = new JfrogClient({
     password: 'password',
     // OR
     accessToken: 'accessToken',
+
+    // Optional parameters
     proxy: {host: '<organization>-xray.jfrog.io', port: 8081, protocol: 'https'},
-    headers: {'key1': 'value1', 'key2': 'value2'}
+    headers: {'key1': 'value1', 'key2': 'value2'},
+    retries: 3
```

Comment:
Let's add the following description for this new property -
// Connection retries. If not defined, the default value is 3.